dom0/securedrop-launcher.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Terminal=false
+Icon=/usr/share/securedrop/icons/sd-logo.png
+Name=SecureDrop Workstation Launcher
```

Comment:
@ninavizz and I would recommend changing this icon label simply to "SecureDrop". The preflight updater will or will not run in some situations.